test-in-qmlscene.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+previewResolutions=( 182 160 144 128 112 )
```

Comment:
I’m not sure how I feel about using 40% of the screen. This sounds like a minor size optimization. It’s fine, we can use a couple more MB if it makes the UI feel sharper.
If you tell me that the watchfaces preview look super sharp at these resolutions then okay fair enough. Otherwise I’d recommend we don’t over-optimize for space. Disk space has never been an issue for Asteroid so far. As long as we stay somewhat sane about our disk usage, it’s okay. Also eventually I’d want these to be generated on the watch itself so we will only have the required resolution, that will be a more significant saving.

Reply:
> If you tell me that the watchfaces preview look super sharp at these resolutions then okay fair enough. 
That is one major advantage of supplying a pre rendered image per resolution. Mogrify is hard to beat with its adaptive resharpen. Compared to the Qt loaded and rendered live view the difference is day and night. Due to Qt badly handling the anti aliasing on those smal resolutions. Figure attached. But that only to show why preview images have a visual advantage over current vanilla by a long shot. 
Another one being, if we supply only one preview image at, lets say 182px and let the watches downscale them.
Those watches with lower resolution and afaiu inferior SoC will have to do scaling work where the faster and higher resolution watches can just use the 182px versions.
Performance difference from just loading the pre renderd images to do any kind of scaling work was striking in my tests.

Left, the preview. Right, the qml live view.
![Bildschirmaufnahme vom 2022-01-12 12:40:23](https://user-images.githubusercontent.com/15074193/149133939-6b4e31f8-1847-4608-aa77-1ff90e91f7ea.png)

Reply:
Reading again, i am not sure if i answered your question.

Reply:
Okay I was assuming the preview was taking 50% of the screen space.
+ yes the quality seems pretty much equivalent. Let's go with this. Thank you